cmd/kpod/main.go

```diff
@@ -57,6 +58,10 @@ func main() {
 			Usage: "path to the 'run directory' where all state information is stored",
 		},
 		cli.StringFlag{
+			Name:  "runtime",
+			Usage: "path to the OCI-compatible binary used to run containers, default is /usr/bin/runc",
```

Comment:
Not crazy about the hard coding here, I think this path should be available in libkpod.  But fine for now.